es6-shim.js

```diff
@@ -1498,7 +1498,11 @@
     },
 
     log2: function (value) {
-      return Math.log(value) * Math.LOG2E;
+      if (value === 0) {
+        return -1 / 0;
```

Comment:
Can this just be `-Infinity`? It's a keyword, so there's no advantage to the division by zero.